database.cnf

```diff
@@ -0,0 +1,10 @@
+[client]
+host = tools.db.svc.eqiad.wmflabs
+database = s53701__worklist
+password = <<PASSWORD>>
+user = <<USER ID>>
+
+[mysqld]
+innodb_file_format = Barracuda
+innodb_file_per_table = 1
+innodb_large_prefix = 1
```

Comment:
Do we need these, and why? Or were they just part of the workarounds we tried before for the column lengths in MariaDB?

If we need them, it's worth adding a comment explaining why.

Reply:
When I was working on this issue, these were the first steps that had to be done. Reference link: https://stackoverflow.com/questions/30761867/mysql-error-the-maximum-column-size-is-767-bytes.
The last step was to set the row_format as Dynamic which had to be done manually for every step. 
If we wouldn't have found a workaround by tweaking social_django variables, the aforementioned step still had to be done. Hence, I didn't remove these lines.

Reply:
Hmm, so this is part of the solution we would have tried if we hadn't found the workaround, correct?

If so, I just don't see why we'd keep this, as it's not really solving any problem right now. Having a half-implemented solution in the code like this just makes things harder to understand (consider coming back and seeing this 6 months for now -- would you remember what it is and what it does?) and sets us apart from what other tools are doing (presumably most just use  the defaults), which can create our own obscure problems.

If you want to implement the solution in the StackOverflow link once Toolforge updates their MariaDB, that's totally fine, and it might even make sense to write down that link in the other comment where you mention the Phabricator ticket, but I don't see why we'd keep part of the solution there now.